Mathlib/Tactic/ToAdditive.lean

```diff
@@ -1,1496 +1,15 @@
 /-
-Copyright (c) 2017 Mario Carneiro. All rights reserved.
+Copyright (c) 2024 Miyahara Kō. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Mario Carneiro, Yury Kudryashov, Floris van Doorn, Jon Eugster
+Authors: Miyahara Kō
 -/
-import Mathlib.Init.Data.Nat.Notation
-import Mathlib.Data.String.Defs
-import Mathlib.Data.Array.Defs
-import Mathlib.Lean.Expr.ReplaceRec
-import Mathlib.Lean.EnvExtension
-import Mathlib.Lean.Meta.Simp
-import Lean.Elab.Tactic.Ext
-import Lean.Meta.Tactic.Symm
-import Lean.Meta.Tactic.Rfl
-import Batteries.Lean.NameMapAttribute
-import Batteries.Tactic.Lint -- useful to lint this file and for for DiscrTree.elements
-import Mathlib.Tactic.Relation.Trans -- just to copy the attribute
-import Mathlib.Tactic.Eqns -- just to copy the attribute
-import Mathlib.Tactic.Simps.Basic
+import Mathlib.Tactic.ToAdditive.Frontend
```

Comment:
Can you please undo the changes to this file. Maybe this is a good change, but it should be a separate PR.

Reply:
#14750